tests/testthat/test-Rules-methods.R

```diff
@@ -735,7 +735,60 @@ test_that("nextBest-NextBestMaxGainSamples returns expected values of the object
   expect_identical(result[names(expected)], expected, tolerance = 10e-7)
 })
 
-# maxDose-IncrementsNumDoseLevels ----
+# maxDose ----
+
+## IncrementsRelative ----
+
+test_that("IncrementsRelative works correctly for last dose in 1st interval", {
+  increments <- IncrementsRelative(intervals = c(0, 110), increments = c(1, 0.5))
+  data <- h_get_data(placebo = FALSE) # The last dose is 100.
```

Comment:
oops not sure why I did not save my comment sorry @wwojciech !
My comment was here about the `data` preparation: it would be more transparent and easy to read if you directly create the `Data()` call here incl. the last dose of 100, then also no comment needed anymore. especially for these `maxDose()` method tests. otherwise is fine to use these data helper constructors.

Reply:
Sure, thanks @danielinteractive, I will do that. The only one small drawback is that it will be a little bit more lines of the code, but I guess it is worth to pay this price.

Reply:
yeah just keep the `Data()` call as minimal as possible. e.g. just one cohort at dose 100 here will be sufficient

Reply:
yap, maybe two cohorts with two different doses - still the code it short but we can test when there are two doses so that we make sure the last one is chosen. Ok, I pushed the changes so you can have a look.